ChangeLog.d/psa_ecc_macros.txt

```diff
@@ -0,0 +1,4 @@
+API changes
+   * Rename PSA_ECC_CURVE_xxx to PSA_ECC_FAMILY_xxx in line with PSA spec
```

Comment:
Please don't use unnecessary abbreviations like “spec” for “specification”.

Reply:
Ok, fixed in comments as well.